conf/kubernetes-shuffle-service.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  labels:
+    app: spark-shuffle-service
+    version: 2.1.0
+  name: shuffle
+spec:
+  template:
+    metadata:
+      labels:
+        app: spark-shuffle-service
+    spec:
+      volumes:
+        - name: temp-volume
+          hostPath:
+            path: '/tmp'
```

Comment:
> By "overhead", do you mean the "setup steps" such as downloading helm and doing $ helm init? Or do you mean the "runtime" cpu, etc overhead?

Yes. Helm is one package manager, but not necessarily installed on most clusters. It would be excessive to expect installations of helm to run a sub-component of Spark in my opinion.

> Independent of helm or not, I still think putting /tmp here is a not a good example.

Fair enough, I'll include a better example, of multiple directories then. What do you think would be a good default shuffle directory on hosts? `/shuffle-files` or something?